test/validate/linux_validate.go

```diff
@@ -138,6 +143,25 @@ func cnsManagedStateFileIps(result []byte) (map[string]string, error) {
 	return cnsPodIps, nil
 }
 
+func cnsManagedStateFileDualStackIps(result []byte) (map[string]string, error) {
```

Comment:
I think we will need a similar func for dualstack cilium ips

Reply:
Updated the struct to take ipv4 and ipv6 since that how they appear in the statefile and created the new function